source/common/stream_info/stream_info_impl.h

```diff
@@ -339,6 +355,8 @@ struct StreamInfoImpl : public StreamInfo {
   absl::optional<Upstream::ClusterInfoConstSharedPtr> upstream_cluster_info_;
   std::string filter_chain_name_;
   Tracing::Reason trace_reason_;
+  BytesMetererSharedPtr upstream_bytes_meterer_ = std::make_shared<BytesMeterer>();
+  BytesMetererSharedPtr downstream_bytes_meterer_ = std::make_shared<BytesMeterer>();
```

Comment:
I don't think these should be default constructed. Since they're being set from the streams' byte meterer.

Reply:
Fixed.